runhouse/servers/cluster_servlet.py

```diff
@@ -465,6 +462,39 @@ def _generate_logs_file_name(self):
         current_timestamp = datetime.datetime.utcnow().strftime("%Y_%m_%d_%H_%M_%S")
         return f"{current_timestamp}_{SERVER_LOGS_FILE_NAME}"
 
+    def call_den_with_post_logs(
```

Comment:
```suggestion
    def send_cluster_logs_to_den(
```

Reply:
called in send_cluster_logs_to_den_helper, cause we are already have a function called send_cluster_logs_to_den, this is being called by the periodic loop that sends cluster status and logs to den.